code/__HELPERS/unsorted.dm

```diff
@@ -1297,31 +1297,6 @@ proc/atan2(x, y)
 	if(!x && !y) return 0
 	return y >= 0 ? arccos(x / sqrt(x * x + y * y)) : -arccos(x / sqrt(x * x + y * y))
 
-proc/rotate_icon(file, state, step = 1, aa = FALSE)

```

Comment:
This was completely unused, and was the second place the ugly variable format was used.